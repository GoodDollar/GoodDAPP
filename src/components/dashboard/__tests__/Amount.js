// eslint-disable-next-line import/order
import React from 'react'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'
import { getWebRouterComponentWithMocks } from './__util__'

jest.setTimeout(30000)

describe('Amount', () => {
  const Amount = getWebRouterComponentWithMocks('../Amount')

  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<Amount />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
