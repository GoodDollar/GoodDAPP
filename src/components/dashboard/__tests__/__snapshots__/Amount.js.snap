// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amount matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-display-6koalj r-flexGrow-16y2uox r-height-1pi2tsx r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-display-6koalj r-flexDirection-eqz5dr r-flexGrow-16y2uox r-maxHeight-87ditf r-overflowX-lltvgl r-overflowY-1rnoaur r-padding-edyy15 r-position-bnwqim r-width-13qz1uu"
      data-name="viewWrapper"
      style={
        Object {
          "backgroundImage": "linear-gradient(3.141592653589793rad,#00AFFF,#2DC0F7,#28C0EF,#23C0E7,#1EC1DF,#19C1D7,#14C1CF,#0FC2C7,#0FC2C7,#0AC2BF,#05C2B7,#00C3AF)",
        }
      }
    >
      <div
        className="css-view-1dbjc4n r-backgroundColor-14lw9ot r-borderRadius-1f0042m r-height-yivrw8 r-justifyContent-1777fci r-marginBottom-5oul0u r-paddingBottom-xd6kpl r-paddingHorizontal-1e081e0 r-paddingLeft-f727ji r-paddingRight-1pyaxff r-paddingTop-tskmnb r-paddingVertical-1yzf0co"
      >
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
        >
          <div
            className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-1it3c9n r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
            dir="auto"
          >
            <div
              className="css-view-1dbjc4n r-alignItems-obd0qt r-display-6koalj r-flexDirection-18u37iz r-justifyContent-17s6mgv"
            >
              <span
                className="css-text-901oao css-textHasAncestor-16my406 r-color-181bzza r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-marginRight-a5pmau r-textAlign-1ff274t r-textDecorationLine-13wfysu r-textTransform-3twk1y"
              >
                0
              </span>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-display-xoduu5 r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
              >
                <span
                  className="css-text-901oao css-textHasAncestor-16my406 r-color-181bzza r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-1ff274t r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                >
                  G
                </span>
                <span
                  className="css-text-901oao css-textHasAncestor-16my406 r-color-181bzza r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-1ff274t r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                >
                  $
                </span>
              </div>
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            onClick={[Function]}
            onContextMenu={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "WebkitTransitionDuration": "0s",
                "transitionDuration": "0s",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-marginRight-61z16t"
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1uqle4r r-borderRadius-ieixw r-display-6koalj r-height-1g7fiml r-justifyContent-1777fci r-marginBottom-d0pm55 r-width-199wky7"
              >
                <div
                  className="css-text-901oao r-fontWeight-1it3c9n"
                  dir="auto"
                  style={
                    Object {
                      "color": "rgba(255,255,255,1.00)",
                      "fontFamily": "gooddollar",
                      "fontSize": "22px",
                      "fontStyle": "normal",
                    }
                  }
                >
                  î ­
                </div>
              </div>
              <div
                className="css-text-901oao r-color-waoq6i r-fontFamily-vnw8o6 r-fontSize-10x49cs r-fontWeight-majxgm r-lineHeight-1p6fy00 r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                dir="auto"
              >
                Scan QR Code
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-backgroundColor-14lw9ot r-borderRadius-1f0042m r-flexGrow-16y2uox r-marginBottom-p1pxzi r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
      >
        <div
          className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-flexGrow-16y2uox"
          style={
            Object {
              "WebkitBoxPack": "start",
              "WebkitJustifyContent": "flex-start",
              "justifyContent": "flex-start",
              "msFlexPack": "start",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-display-6koalj r-flex-13awgt0 r-justifyContent-1wtj0ep r-width-13qz1uu"
          >
            <div
              className="css-view-1dbjc4n r-height-1w2pmg r-minHeight-ifefl9"
            >
              <div
                className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-hiw28u r-fontWeight-majxgm r-lineHeight-qvk6io r-marginVertical-c8eef1 r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-tsynxw"
                dir="auto"
              >
                How much?
              </div>
              <div
                className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
              >
                <div
                  className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
                >
                  <input
                    autoCapitalize="sentences"
                    autoComplete="on"
                    autoCorrect="on"
                    autoFocus={true}
                    className="css-textinput-11aywtz r-backgroundColor-14lw9ot r-borderBottomColor-1htqvjr r-borderBottomWidth-qklmqi r-color-13sdq65 r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-vw2c0b r-letterSpacing-19r33im r-marginTop-1mnahxq r-paddingHorizontal-1dye5f7 r-paddingVertical-oyd9sg r-placeholderTextColor-1xktb6b r-textAlign-q4m81j"
                    dir="auto"
                    maxLength={20}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onClick={[Function]}
                    onContextMenu={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onSelect={[Function]}
                    onTouchStart={[Function]}
                    placeholder="0 DEVELOP"
                    readOnly={false}
                    spellCheck={true}
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              className="css-view-1dbjc4n r-display-6koalj r-flexGrow-16y2uox r-justifyContent-1wtj0ep r-marginBottom-1ifxtd0 r-marginTop-1wzrnnt r-maxHeight-jtcya5 r-overflow-1udh08x"
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    1
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    2
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    3
                  </div>
                </div>
              </div>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    4
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    5
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    6
                  </div>
                </div>
              </div>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    7
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    8
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    9
                  </div>
                </div>
              </div>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    .
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-text-901oao r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-hiw28u r-fontWeight-b88u0q r-lineHeight-qvk6io r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                    dir="auto"
                  >
                    0
                  </div>
                </div>
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-cursor-1loqt21 r-display-6koalj r-flex-13awgt0 r-padding-edyy15 r-touchAction-1otgn73 r-userSelect-lrvibr"
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onKeyDown={[Function]}
                  tabIndex="0"
                >
                  <div
                    className="css-view-1dbjc4n r-backgroundRepeat-u6sd8q r-backgroundSize-ehq7j7 r-cursor-1loqt21 r-height-1g7fiml r-justifyContent-1777fci r-width-1aockid"
                  >
                    <div
                      className="css-text-901oao r-color-cddv6f r-fontWeight-1it3c9n r-textAlign-q4m81j"
                      dir="auto"
                      style={
                        Object {
                          "fontFamily": "gooddollar",
                          "fontSize": "18px",
                          "fontStyle": "normal",
                        }
                      }
                    >
                      î 
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1wtj0ep"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-flexGrow-16y2uox"
            style={
              Object {
                "WebkitBoxPack": "start",
                "WebkitJustifyContent": "flex-start",
                "justifyContent": "flex-start",
                "msFlexPack": "start",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1oszu61 r-display-6koalj r-justifyContent-1777fci r-marginBottom-p1pxzi r-marginRight-61z16t r-marginTop-1mnahxq"
              style={
                Object {
                  "backgroundColor": "rgba(0,0,0,0.00)",
                  "borderBottomColor": "rgba(0,175,255,1.00)",
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "0px",
                  "borderLeftColor": "rgba(0,175,255,1.00)",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "0px",
                  "borderRightColor": "rgba(0,175,255,1.00)",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "0px",
                  "borderTopColor": "rgba(0,175,255,1.00)",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "0px",
                  "marginLeft": "0px",
                  "minHeight": "44px",
                  "minWidth": "auto",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                }
              }
            >
              <div
                className="css-cursor-18t94o4 css-view-1dbjc4n r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim"
                onClick={[Function]}
                onContextMenu={[Function]}
                onKeyDown={[Function]}
                role="button"
                style={
                  Object {
                    "WebkitAlignSelf": "stretch",
                    "alignSelf": "stretch",
                    "borderBottomLeftRadius": "50px",
                    "borderBottomRightRadius": "50px",
                    "borderTopLeftRadius": "50px",
                    "borderTopRightRadius": "50px",
                    "msFlexItemAlign": "stretch",
                    "msGridRowAlign": "stretch",
                  }
                }
                tabIndex="0"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
                  style={
                    Object {
                      "WebkitAlignSelf": "stretch",
                      "alignSelf": "stretch",
                      "msFlexItemAlign": "stretch",
                      "msGridRowAlign": "stretch",
                    }
                  }
                >
                  <div
                    className="css-view-1dbjc4n r-marginHorizontal-1ybube5 r-marginVertical-712mwr"
                  >
                    <div
                      className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flex-13awgt0 r-flexDirection-18u37iz"
                    >
                      <div
                        className="css-text-901oao r-alignItems-1awozwy r-color-vaa0vc r-display-6koalj r-fontFamily-vnw8o6 r-fontSize-1b43r93 r-fontWeight-majxgm r-justifyContent-1777fci r-letterSpacing-oxtfae r-lineHeight-qvk6io r-minHeight-1ceczpf r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                        dir="auto"
                      >
                        Cancel
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-justifyContent-1wtj0ep r-minWidth-bcqeeo"
            style={
              Object {
                "WebkitBoxFlex": 3,
                "WebkitFlexGrow": 3,
                "flexGrow": 3,
                "msFlexPositive": 3,
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1oszu61 r-display-6koalj r-justifyContent-1777fci r-marginBottom-p1pxzi r-marginRight-61z16t r-marginTop-1mnahxq"
              style={
                Object {
                  "WebkitFlex": 2,
                  "backgroundColor": "rgba(23,53,102,0.40)",
                  "borderBottomColor": "rgba(0,175,255,1.00)",
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "0px",
                  "borderLeftColor": "rgba(0,175,255,1.00)",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "0px",
                  "borderRightColor": "rgba(0,175,255,1.00)",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "0px",
                  "borderTopColor": "rgba(0,175,255,1.00)",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "0px",
                  "flex": 2,
                  "marginLeft": "0px",
                  "minHeight": "44px",
                  "minWidth": "64px",
                  "msFlex": "2 1 0%",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                }
              }
            >
              <div
                aria-disabled={true}
                className="css-view-1dbjc4n r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim"
                onClick={[Function]}
                onContextMenu={[Function]}
                onKeyDown={[Function]}
                role="button"
                style={
                  Object {
                    "WebkitAlignSelf": "stretch",
                    "alignSelf": "stretch",
                    "borderBottomLeftRadius": "50px",
                    "borderBottomRightRadius": "50px",
                    "borderTopLeftRadius": "50px",
                    "borderTopRightRadius": "50px",
                    "msFlexItemAlign": "stretch",
                    "msGridRowAlign": "stretch",
                  }
                }
                tabIndex="-1"
              >
                <div
                  className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
                  style={
                    Object {
                      "WebkitAlignSelf": "stretch",
                      "alignSelf": "stretch",
                      "msFlexItemAlign": "stretch",
                      "msGridRowAlign": "stretch",
                    }
                  }
                >
                  <div
                    className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
                  >
                    <div
                      className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flex-13awgt0 r-flexDirection-18u37iz"
                    >
                      <div
                        className="css-text-901oao r-alignItems-1awozwy r-color-jwli3a r-display-6koalj r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-justifyContent-1777fci r-letterSpacing-oxtfae r-lineHeight-qvk6io r-minHeight-1ceczpf r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-tsynxw"
                        dir="auto"
                      >
                        Next
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
